### PR TITLE
PMM-7015 Use read values from get responce.

### PR DESCRIPTION
**Pull request by askomorokhov**
[PMM-7015](https://jira.percona.com/browse/PMM-7015)

- [x] Build: https://github.com/Percona-Lab/pmm-submodules/pull/1375

- [x] https://github.com/percona-platform/dbaas-controller/pull/117
---
- [ ] Tests passed.
- [ ] Feature build pass.
- [ ] (Re)requested review.
- [ ] Fix conflicts with target branch.
- [ ] Update API dependency.
- [ ] Update jira ticket description if needed.
- [ ] Attach screenshots/console output to confirm new behavior to jira ticket, if applicable.

### CODE REVIEW

**Comment by JiriCtvrtka** [comment]
@askomorokhov Build failing.

File: services/management/dbaas/psmdb_cluster_service.go [hunk 1]
```diff
@@ -130,10 +125,10 @@ func (s PSMDBClusterService) GetPSMDBCluster(ctx context.Context, req *dbaasv1be
 
 	resp := dbaasv1beta1.GetPSMDBClusterResponse{
 		ConnectionCredentials: &dbaasv1beta1.GetPSMDBClusterResponse_PSMDBCredentials{
-			Username:   "userAdmin",
-			Password:   "userAdmin123456",
+			Username:   cluster.Credentials.Username,
+			Password:   cluster.Credentials.Password,
 			Host:       host,
```
Comment (BupycHuk) [inline services/management/dbaas/psmdb_cluster_service.go:130]:
```suggestion
			Host:       cluster.Credentials.Host,
```

File: services/management/dbaas/xtradb_cluster_service.go [hunk 2]
```diff
@@ -139,10 +134,10 @@ func (s XtraDBClusterService) GetXtraDBCluster(ctx context.Context, req *dbaasv1
 	_ = kubernetesCluster
 	resp := dbaasv1beta1.GetXtraDBClusterResponse{
 		ConnectionCredentials: &dbaasv1beta1.XtraDBClusterConnectionCredentials{
-			Username: "root",
-			Password: "root_password",
+			Username: cluster.Credentials.Username,
+			Password: cluster.Credentials.Password,
 			Host:     host,
```
Comment (BupycHuk) [inline services/management/dbaas/xtradb_cluster_service.go:139]:
```suggestion
			Host:     cluster.Credentials.Host,
```

**Review by BupycHuk — APPROVED** [review]

**Review by BupycHuk — CHANGES_REQUESTED** [review]
Tests are failing, and could you change base branch to `release/2.14`

**Review by JiriCtvrtka — APPROVED** [review]

**Review by BupycHuk — APPROVED** [review]

**Pull request merged**